cli/lib/check_deps.go

```diff
@@ -51,13 +51,13 @@ func runDepsCommand(cmd *cobra.Command, args []string) {
 		os.Exit(errorcodes.ErrBadArgument)
 	}
 
-	deps, err := lib.LibraryResolveDependencies(context.Background(), &rpc.LibraryResolveDependenciesRequest{
+	deps, stat := lib.LibraryResolveDependencies(context.Background(), &rpc.LibraryResolveDependenciesRequest{
```

Comment:
`stat` is of type `error` we should probably rename it `err`, the name is kinda confusing otherwise.